tests/test_vqe.py

```diff
@@ -886,6 +887,38 @@ def circuit(params):
         mt2 = circuit.metric_tensor([p])
         assert np.allclose(mt, mt2)
 
+    def test_multiple_devices(self, mocker):
+        """Test that passing multiple devices to ExpvalCost works correctly"""
+
+        dev = [qml.device("default.qubit", wires=2), qml.device("default.mixed", wires=2)]
+        spy = mocker.spy(DefaultQubit, "apply")
+        spy2 = mocker.spy(DefaultMixed, "apply")
+
+        h = qml.Hamiltonian([1, 1], [qml.PauliZ(0), qml.PauliZ(1)])
+
+        qnodes = qml.ExpvalCost(qml.templates.StronglyEntanglingLayers, h, dev)
+        w = qml.init.strong_ent_layers_uniform(3, 2, seed=1967)
+
+        res = qnodes(w)
+        exp = 0.3864767030806935
```

Comment:
How did you determine this value? Looks a bit 'magic'.

Reply:
Made it clearer now, thanks!